client/vault/smoke_test.py

```
@@ -0,0 +1,1078 @@
+
+
+def test_imports():
+    """Test that all expected classes are importable."""
+    print("✓ All imports successful")
+
+
+def test_key_credential_creation():
+    """Test KeyCredential dataclass creation."""
+    cred = KeyCredential(
+        account_id=AccountId("test.near"),
+        secret_key="ed25519:3D4YudUQRE39Lc4JHghuB5WM8kbgDDa34mnrEP5DdTApVH81af3e7MvFrog1CMNn67PCi6eC8x9TgDxCV9ySeGir",
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)